app/controllers/contacts_controller.rb

```diff
@@ -9,7 +9,11 @@ def index
 
   def search_params
     filter = { organisation_id: organisation.id }
-    filter.merge(params.fetch(:search, {}))
+    filter = filter.merge(params.fetch(:search, {}))
+    # rewrite requests using older search parameter name
+    filter[:q] = filter['name']
```

Comment:
Minor point - can't this and the next line be combined to be `filter[:q] = filter.delete('name')`?